### PR TITLE
docs: add in docs about doctorVisibilityDidChange

### PR DESCRIPTION
**Pull request by ckipp01**

### CODE REVIEW

**Review by dos65 — COMMENTED** [review]
:+1:

File: docs/integrations/new-editor.md [hunk 1]
```diff
@@ -1062,3 +1062,21 @@ interface FindTextInDependencyJarsRequest(
 _Response_:
 
 - result: `Location[]`
+
+### `metals/doctorVisibilityDidChange`
+
+When the client opens the doctor view, it will send in `visible: true` to the
```
Comment (kpodsiad) [inline docs/integrations/new-editor.md:1068]: out of curiosity, is `in` needed in `send in to the server`?

**Review by kpodsiad — APPROVED** [review]

File: docs/integrations/new-editor.md (hunk 1 above)
Comment (ckipp01) [reply docs/integrations/new-editor.md:1068]: ha, funny questions. Both would be correct. They essentially mean exactly the same thing in this scenario. I can remove it if you think it reads better.

Comment (kpodsiad) [reply docs/integrations/new-editor.md:1068]: Lets keep it, I was just curious ;)

**Pull request merged**